ophyd/epics_motor.py

```diff
@@ -145,8 +145,18 @@ def move(self, position, wait=True, **kwargs):
             If motion fails other than timing out
         '''
         self._started_moving = False
+        status = None
 
+        def dmov_callback(old_value, value, **kwargs):
+            self.log.debug(f'DEBUG: dmov_callback(): new={value} old={old_value}')
```

Comment:
```suggestion
            self.log.debug('dmov_callback(): new=%f, old=%f', value, old_value)
```
Two things here:
- The logging formatter will add the "DEBUG:" part for us; it shouldn't be in the message body. (It's very possible we aren't consistent about this in ophyd. The logging overall needs a thorough review.)
- A bit of Python logging best practice... Although we love and encourage f-strings in general, for log messages in particular the old-style string templating is recommended because the logging framework can ensure that the templating will only be performed if that log message is actually issued. That is, if there is nothing handling DEBUG-level log messages, then you won't pay the performance overhead of building this string. Note that the values are passed as arguments to `debug`, like `debug('...', a, b)` not `'debug(...' % (a, b))`. The formatting is deferred and only happens if it needs to. "If a {Python log message} falls in a forest and there's no one there to hear it...."

Reply:
I hear you.  `DEBUG: ` is out and back to old style strings.